### DISCUSSION TITLE
MITRE ATT&CK Navigator and  SO dashboard issue

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#10279] [2.4] [answered]
ERROR parsing layer from assets/playbook.json, check the javascript console for more information.
![화면 캡처 2023-05-04 205821](https://user-images.githubusercontent.com/96872399/236197250-24626aa8-b9c8-45af-9b5a-cca2d1587cdd.png)

Request failed with status code 404
See the Help section of the Security Onion documentation for additional troubleshooting guidance.
![화면 캡처 2023-05-04 205605](https://user-images.githubusercontent.com/96872399/236196784-4f1871c0-0175-4aaa-833f-c3c2bb37d803.png)

I’m sorry to bother you, but I don’t know where to look at the logs of these services. Could you please tell me where they are located and I will promptly send them to you?

**Comment by dougburks** [comment, marked as answer]
For your first screenshot, please see the Known Issues section of the blog post at https://blog.securityonion.net/2023/04/security-onion-24-beta-2-release-now.html:
```
ATT&CK Navigator doesn’t work correctly yet.
```

Your second screenshot shows that you are running version 2.4.2 which has not been released yet. This would imply that you installed from our development branch (`2.4/dev`) instead of the `2.4/main` branch as shown in the installation instructions. From https://docs.securityonion.net/en/2.4/installation.html#installation-on-rocky-linux-or-ubuntu:
```
git clone -b 2.4/main https://github.com/Security-Onion-Solutions/securityonion
```

The SOC error should not exist in the `2.4/main` branch (version 2.4.1) and should be resolved in version 2.4.2 before it is fully released.
